tests/unit/ext/switches/brocade.py

```diff
@@ -3,12 +3,31 @@
 import pytest
 import requests_mock
 
-from hil import model
-from hil.test_common import fail_on_log_warnings
+from hil import model, config
+from hil.test_common import fail_on_log_warnings, config_testsuite, \
+ config_merge
 
 fail_on_log_warnings = pytest.fixture(autouse=True)(fail_on_log_warnings)
 
 
+@pytest.fixture
+def configure():
+    """Configure HIL"""
+    config_testsuite()
+    config_merge({
+        'extensions': {
+            'hil.ext.network_allocators.null': None,
+            'hil.ext.network_allocators.vlan_pool': '',
+        },
+        'hil.ext.network_allocators.vlan_pool': {
+            'vlans': '100-105',
+        },
+    })
+    config.load_extensions()
```

Comment:
What changed to make this necessary?

Reply:
in modify port I am using `get_network_allocator().is_legal_channel_for()` to assert that vlan_id is valid for channel. I remember [you suggested](https://github.com/CCI-MOC/hil/pull/866#discussion_r137414606) during the review of the dellnos9 driver. 

So, without this change, if there's no network allocator then it wouldn't work.

Reply:
Aha, makes sense.